### PR TITLE
Add pt_BR translations

### PR DESCRIPTION
**Pull request by waldyr**

### CODE REVIEW

File: war/src/main/webapp/help/view-config/filter-queue_pt_BR.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-2009, Sun Microsystems, Inc., Tom Huybrechts
```
Comment (daniel-beck) [inline war/src/main/webapp/help/view-config/filter-queue_pt_BR.html]: Nope.

File: war/src/main/webapp/help/view-config/filter-executors_pt_BR.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-2009, Sun Microsystems, Inc., Tom Huybrechts
```
Comment (daniel-beck) [inline war/src/main/webapp/help/view-config/filter-executors_pt_BR.html]: Nope.

**Comment by daniel-beck** [comment]
@kinow Does this look correct?

**Comment by waldyr** [comment]
I took those from the original file. Maybe it's time to change them all. What should be the replacement?

**Comment by daniel-beck** [comment]
@waldyr 

> I took those from the original file. Maybe it's time to change them all. What should be the replacement?

Do not change the existing files. Use your name in place of Sun's/Tom's.

**Comment by kinow** [comment]
@boaglio want to review it together? I remember we had a list of items to avoid translating, but I'll have to find it.

**Comment by waldyr** [comment]
I would love to have that file, because I'm planning to make more pt_BR translations. Am I allowed to use my company (Code Miner 42) instead of Sun? It would end up like the following:

```
Copyright (c) 2004-2016, Codeminer42, Waldyr de Souza
```

**Comment by boaglio** [comment]
yes @kinow  =)

@waldyr   thank you for you help, please review these items:

We have some terms that we don't translate, like pipeline or job. 

Please take a look: 
https://wiki.jenkins-ci.org/display/JENKINS/Brazilian+Portuguese+Translation

tarefa => job
tarefas => jobs

- _patternJobNamingStrategy_pt_BR.html_
em tarefas que já existe, => em jobs que já existem,
A tarefa vai ser validada => O job será validado

- _help-label_pt_BR.html_
 irão ser  =>  serão

- _tool-location-node-property_pt_BR.html_
automaticos => automáticos

- _ilter-executors_pt_BR.html_
podem executar tarefas nesta view => podem executar jobs nessa view 

- _ filter-queue_pt_BR.html_
apenas tarefas nesta view => apenas jobs nessa view

**Comment by kinow** [comment]
:clap: @boaglio thanks for chiming in!!!

**Comment by waldyr** [comment]
@boaglio @kinow @daniel-beck Everything was addressed as requested 👍

**Comment by boaglio** [comment]
great job @waldyr   :+1:

**Comment by daniel-beck** [comment]
Thanks everyone!

**Pull request merged**